### PR TITLE
CRPL-2102: Implements a node for reading list of environment variable…

### PR DESCRIPTION
**Pull request by gyhevizi**
Adds node that accepts a list of environment variable names and sends a dictionary with those that could be resolved from either env variables or from provided defaults.

### CODE REVIEW

**Review by csabee1979 — APPROVED** [review]

File: system/environment variables getter with fallback/2ac2a613-e930-4e37-9a28-02ce0421836b.json [hunk 1]
```diff
@@ -0,0 +1,186 @@
+{
+  "name": "system/Environment variables getter with fallback",
+  "interface": {
+    "inputs": {
+      "a51796cd-95b6-43e6-88e8-54ae80e59646": {
+        "type": "string[]",
+        "name": "variable names",
+        "description": "Receives a list of variable names to be resolved from the environment.\n\nExample:\n[\"Var1\", \"Var2\"]"
```
Comment (lvass74) [inline system/environment variables getter with fallback/2ac2a613-e930-4e37-9a28-02ce0421836b.json:8]: Please add tag to the example, like described here: https://kwaia.atlassian.net/wiki/spaces/CRANQ/pages/804356131/CRANQ+Component+Documentation#Examples
Something like: 
`["Var1", "Var2"]@0`

File: system/environment variables getter with fallback/2ac2a613-e930-4e37-9a28-02ce0421836b.json [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+      },
+      "2793a39a-95ef-44bd-a075-6554a4c35d0b": {
+        "type": "{string:string}",
+        "name": "default values",
+        "description": "Receives optional default values for undefined environment variables.\n\nExample:\n{ \"var1\": \"value1\" }"
```
Comment (lvass74) [inline system/environment variables getter with fallback/2ac2a613-e930-4e37-9a28-02ce0421836b.json:13]: Please add tag to example

File: system/environment variables getter with fallback/2ac2a613-e930-4e37-9a28-02ce0421836b.json [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+      }
+    },
+    "outputs": {
+      "25b50086-6321-402e-82e6-8b7879683cc3": {
+        "type": "{string:string}",
+        "name": "resolved variables",
+        "description": "Sends the dictionary of resolved environment variables as a name:value dictionary.\n\nContains the environment value if found, otherwise the specified default value if any.\n\n\nExample:\n{\n  \"V1\": \"value 1\",\n  \"V2\": \"default value 2\"\n}"
```
Comment (lvass74) [inline system/environment variables getter with fallback/2ac2a613-e930-4e37-9a28-02ce0421836b.json:20]: Please add tag to example

File: system/environment variables getter with fallback/2ac2a613-e930-4e37-9a28-02ce0421836b.json [hunk 4]
```diff
@@ -0,0 +1,186 @@
...
+          "25b50086-6321-402e-82e6-8b7879683cc3": {
+            "65aba3c0-6d88-49d3-b961-5f9b09babb38": {
+              "0a9cd5b0-6de6-4c69-aa04-2816de994173": {}
+            }
+          }
+        }
+      }
+    }
+  },
+  "description": "Gets a list variable from either the system environment by name or the provided fallback default values. Sends a dictionary with a dictionary of names that have been resolved either way.",
```
Comment (lvass74) [inline system/environment variables getter with fallback/2ac2a613-e930-4e37-9a28-02ce0421836b.json:181]: Please add general example here too. Like described here: https://kwaia.atlassian.net/wiki/spaces/CRANQ/pages/804356131/CRANQ+Component+Documentation#Examples

**Review by lvass74 — APPROVED** [review]

**Comment by lvass74** [comment]
Some of my above comments are still valid (not resolved) but they are minor issues, so you can go ahead with the merge if needed.

**Pull request merged**